content/en/infrastructure/process/increase_process_retention.md

```diff
@@ -2,6 +2,7 @@
 title: Increase Process Retention
 kind: documentation
 aliases:
+    - /infrastructure/process/increase_process_retention/
```

Comment:
```suggestion
    - /infrastructure/process/generate_process_metrics/
```

Reply:
So question, it looks like this sort of exists down on line 7, but without the "process", and I left it there so that a redirect would be in place. Should it have both, in addition to having an alias for the new page name?

Reply:
Actually, I think the thing I want to ask/got confused about is this: Does this section of the front matter require a line for the _existing_ page name/location? Or just past ones/ones we might have links pointing to?